test/y2storage/initial_guided_proposal_multidisk_first_test.rb

```diff
@@ -0,0 +1,526 @@
+                proposal.propose
+
+                expect(created_vgs_names).to contain_exactly("system")
+              end
+
+              it "creates the proposal using only one device" do
+                proposal.propose
+
+                expect(used_devices).to contain_exactly("/dev/sdb")
+              end
+            end
+
+            context "and it does not fit in a single device" do
+              # The minimal proposal including all volumes needs 30GiB
+              let(:sda_size) { 10.GiB }
+              let(:sdb_size) { 6.GiB }
+              let(:sdc_size) { 6.GiB }
+              let(:sdd_size) { 10.GiB }
+
+              it "creates only the system LVM VG" do
+                proposal.propose
+
+                expect(created_vgs_names).to contain_exactly("system")
+              end
+
+              it "creates the proposal using needed devices" do
+                proposal.propose
+
+                expect(used_devices)
+                  .to contain_exactly("/dev/sda", "/dev/sdb", "/dev/sdc", "/dev/sdd")
+              end
+            end
+          end
+        end
+      end
+
+      context "not using LVM" do
+        let(:lvm) { false }
+
+        context "with separated volume groups" do
+          let(:separate_vgs) { true }
+
+          context "and enough room for a full proposal" do
+            context "and it fits in only one device" do
+              let(:sda_size) { 40.GiB }
+              let(:sdb_size) { 80.GiB }
+              let(:sdc_size) { 20.GiB }
+              let(:sdd_size) { 20.GiB }
+
+              it "creates as partition all volumes with non separate_vg_name" do
+                proposal.propose
+
+                expect(created_mount_paths).to include("/")
+              end
+
+              it "creates as LVM VG all volumes with separate_vg_name" do
+                proposal.propose
+
+                expect(created_vgs_names).to contain_exactly("vg-home", "vg-foobar")
+              end
+
+              # FIXME: it seems that SpaceMaker is having troubles to assign
+              # space at the time to create multiple LVM VGs when
+              # allocate_volume_mode => :auto and separate_vgs => true
+              xit "uses only needed devices" do
+                proposal.propose
+
+                expect(used_devices).to contain_exactly("/dev/sdb")
+              end
+            end
+
+            context "and it does not fit in a single device" do
+              # The minimal proposal including all volumes needs 30GiB
+              let(:sda_size) { 8.GiB }
+              let(:sdb_size) { 8.GiB }
+              let(:sdc_size) { 10.GiB }
+              let(:sdd_size) { 11.GiB }
+
+              # FIXME: it seems that SpaceMaker is having troubles to assign
+              # space at the time to create multiple LVM VGs when
+              # allocate_volume_mode => :auto and separate_vgs => true
+              xit "creates as partition all volumes without separate_vg_name" do
```

Comment:
NP: I would reword it as: "creates partitions for all volumes without separate_vg_name".